scripts/package

```diff
@@ -37,5 +49,5 @@ echo Built ${IMAGE} #${AGENT_IMAGE}
 echo
 
 cd ../bin
-mkdir -p /tmp/system-charts && git clone https://github.com/rancher/system-charts /tmp/system-charts
+checkout-system-charts
```

Comment:
Can we simply do a `git clone --branch master`?

Reply:
Updated